### DISCUSSION TITLE
It is no longer possible to send a request to update the formula

### DISCUSSION THREAD

**Discussion by sandergol** [Homebrew/discussions#2655] [Casks]
After my last request to update the formula https://github.com/Homebrew/homebrew-cask/pull/114667 I can't do it anymore.

I do not have the right to create a branch and there is no way to change the file in the github interface itself.

What happened? How or who updates formulas now?

**Comment by neutric** [comment]
The process hasn't changed. You commit your change to a branch on your fork of `homebrew-cask` and then create a PR from that. What's the error message?

For editing online, go to the repo in question and then press `.` to open the web-based editor.

**Comment by SMillerDev** [comment]
You're supposed to make a change in your own fork and then make a pull request. Nothing changed in that system since you created that pull request.

**Comment by sandergol** [comment]
Do you have a tutorial?

**Reply by neutric** [reply]
https://github.com/Homebrew/homebrew-cask/blob/master/CONTRIBUTING.md

**Reply by neutric** [reply]
Sorry for the very brief previous response. Wasn't meant to have gone out like that.

You have already created [your fork of `homebrew cask`](https://github.com/sandergol/homebrew-cask/branches) and also managed to submit a Pull Request by creating a commit in a branch on your fork. So you must have done the necessary steps as laid out in [CONTRIBUTING.md](https://github.com/Homebrew/homebrew-cask/blob/master/CONTRIBUTING.md) and [adding_a_cask.md](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#submitting-your-changes). This process is not different from what contributions to other projects on GitHub look like.

Hope that careful studies of the above can get things going again! Feel free to ask further questions. If satisfied, please mark this as answered.

**Reply by sandergol** [reply]
Thanks
